docs/components/Wrapper.jsx

```diff
@@ -17,6 +17,13 @@ const styles = {
     marginRight: 0,
     zIndex: 10
   },
+  buttonLang: {
+    position: 'absolute',
+    top: '0.75rem',
+    right: '6.75rem',
```

Comment:
Arbitrary stuff. We should do better with an absolute wrapper & floating buttons inside, but it works (spent 5 minutes on it). We can improve that latter IMHO.